examples/v1/taskruns/beta/stepaction-passing-results.yaml

```diff
@@ -60,7 +60,7 @@ kind: TaskRun
 metadata:
   name: step-action-run
 spec:
-  TaskSpec:
+  taskSpec:
```

Comment:
Is this example not executed in CI, or is it executed but the capital letter version was accepted somehow by Tekton?